tests/by-util/test_nl.rs

```diff
@@ -800,3 +800,102 @@ fn test_file_with_non_utf8_content() {
         String::from_utf8_lossy(invalid_utf8)
     ));
 }
+
+// Regression tests for issue #9132: repeated flags should use last value
+#[test]
+fn test_repeated_body_numbering_flag() {
+    // -ba -bt should use -bt (number all lines)
```

Comment:
The `t` in `-bt` means "number only nonempty lines", and not "number all lines".
```suggestion
    // -ba -bt should use -bt (number only nonempty lines)
```